lib/runtime.js

```diff
@@ -3,17 +3,16 @@
 var Entry = require("./entry.js");
 var utils = require("./utils.js");
 
-exports.enable = function (Module) {
-  var Mp = Module.prototype;
-
-  if (typeof Mp.importSync !== "function" ||
-      typeof Mp.export !== "function") {
-    Mp.runModuleSetters = runModuleSetters;
-    Mp.importSync = moduleImport;
-    Mp.export = moduleExport;
+exports.enable = function (mod) {
```

Comment:
Since this is just bolting things on they could always pass `Module.prototype`. Our runtime stuff isn't really exposed though so I wouldn't want to support it for public mucking.